### PR TITLE
Iroha 2 RC19 update

### PR DESCRIPTION
**Pull request by Mingela**

### CODE REVIEW

**Review by mversic — APPROVED** [review]

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Extensions.kt [hunk 1]
```diff
@@ -88,7 +100,7 @@ fun String.asAssetId() = this.split(ASSET_ID_DELIMITER).takeIf {
     }
 } ?: throw IllegalArgumentException("Incorrect asset ID: $this")
 
-fun String.asTokenId() = PermissionTokenId(Name(this))
+// fun String.asTokenId() = PermissionTokenId(Name(this))
```
Comment (arndey) [inline modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Extensions.kt:103]: I think it should either be commented or deleted

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/query/QueryBuilder.kt [hunk 2]
```diff
@@ -302,15 +294,14 @@ class QueryBuilder<R>(
...
-        @JvmStatic
-        @JvmOverloads
-        fun findAllPermissionTokenDefinitions(
-            queryFilter: GenericPredicateBox<ValuePredicate>? = null,
-        ) = QueryBuilder(
-            Queries.findAllPermissionTokenDefinitions(),
-            PermissionTokenDefinitionsExtractor,
-            queryFilter,
-        )
+//        @JvmStatic
```
Comment (arndey) [inline modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/query/QueryBuilder.kt:297]: uncomment or delete

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 3]
```diff
@@ -1006,6 +1005,18 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
             }
     }
 
+    @Test
+    @WithIroha([WithManyDomains::class])
+    @Story("Check query cursor")
+    @SdkTestId("query_cursor_test")
+    fun `cursor test`(): Unit = runBlocking {
```
Comment (arndey) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt:1012]:
```suggestion
    fun `quering multiple domains with cursor test`(): Unit = runBlocking {
```

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt [hunk 4]
```diff
@@ -1006,6 +1005,18 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
...
+        val domains = QueryBuilder.findAllDomains()
```
Comment (arndey) [inline modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt:1013]: I'd expand this test

Comment (Mingela) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt:1013]: with what exactly? please feel free to commit as well 😃

Comment (Mingela) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt:1013]: agreed to comment about the internal implementation for size/cursor etc

File: modules/test-tools/src/main/resources/.cleanup.wip-7
Comment (Mingela) [inline modules/test-tools/src/main/resources/.cleanup.wip-7:1]: what is this file?

**Review by Mingela — COMMENTED** [review]
LGTM

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 4 above)
Comment (gv-timur) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt:1016]: added comment

File: modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt (hunk 3 above)
Comment (gv-timur) [reply modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt]: fixed

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/query/QueryBuilder.kt (hunk 2 above)
Comment (gv-timur) [reply modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/query/QueryBuilder.kt]: added test

File: modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Extensions.kt (hunk 1 above)
Comment (gv-timur) [reply modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Extensions.kt]: deleted

**Review by arndey — APPROVED** [review]

**Review by gv-timur — APPROVED** [review]

**Pull request merged**